exporter.go

```diff
@@ -266,6 +266,9 @@ func (b *Exporter) Listen(e <-chan Events) {
 				}
 			} else {
 				eventsUnmapped.Inc()
+				if *dropUnmapped == true {
```

Comment:
It's unnecessary and uncommon in Golang to compare a boolean value with a boolean value. Please simplify to `if *dropUnmapped {`.